ui/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala

```diff
@@ -28,7 +28,7 @@ class AppResources(config: Config,
                    jobStatusService: JobStatusService)(implicit ec: ExecutionContext)
   extends Directives with FailFastCirceSupport with LazyLogging with RouteWithUser with RouteWithoutUser with SecurityDirectives {
 
-  //Public app resources
+  //We use duplicate pathPrefix("app") code - look at comment in NussknackerApp where routes are creating
```

Comment:
Can be: "we duplicate ... code" or "we use dulicate*d* ... code" The first one is IMO better .
Also should be: "where routes are creat*ed*"

Reply:
Ups.. Changed.